lib/parallel_tests/test/runner.rb

```diff
@@ -209,16 +209,26 @@ def sort_by_filesize(tests)
         end
 
         def find_tests(tests, options = {})
+          tests_list = all_tests(tests, options)
+          exclude_tests(tests_list, options)
```

Comment:
this seems asymmetric ... do it at the same place the pattern matching is done ?

Reply:
Addressed.